### ISSUE TITLE
Failing tests throw error with Julia v1.10

### ISSUE THREAD

**Issue by mnemnion** [ssfrr/TestSetExtensions.jl#22]
I've tested against 1.9.4, this doesn't happen. 

The error:

```
  MethodError: no method matching scrub_backtrace(::Vector{Union{Ptr{Nothing}, Base.InterpreterIP}})
```
It's coming from `TestSetExtensions.jl:104`.

**Comment by mnemnion** [comment]
The problem is in fact with Test.jl:

```julia 
julia> Test.scrub_backtrace(backtrace())
```
Throws the same error.

**Comment by mnemnion** [comment]
I've found [the problem](https://github.com/JuliaLang/julia/pull/50759/files), and it's better to fix TestSetExtensions, the old `scrub_backtrace` is no longer supported.

**Comment by JTHesse** [comment]
I think the same issue still persist: 
```
using Test
using TestSetExtensions

@testset ExtendedTestSet "All the tests" begin
    @test false
end
```
will get the same:
`MethodError: no method matching scrub_backtrace(::Vector{Union{Ptr{Nothing}, Base.InterpreterIP}})` Error

Tested with Julia 1.10.4 and TestSetExtensions 2.0.0

**Comment by palday** [comment]
@JTHesse It's fixed in TestSetExtensions 3.0.
